src/Components/GetAllReviews.jsx

```diff
@@ -27,7 +29,7 @@ const GetAllReviews = () => {
         <div className="allreviewscontainer">
             <ul className="allreviewslist">
                 {reviews.map((review, index) => {
-                    return <li className="allreviewslistitems" key={index}> {review.title} </li>
+                    return <Link to={`/Review/${review.review_id}`} ><li className="allreviewslistitems" key={index}>"{review.title}" <br />Author: {review.owner} <br />comments: {review.comment_count} <br /> Votes: {review.votes}  </li> </Link>
                 })}
```

Comment:
I'm getting an error for no unique key id here in the console and also for get reviews by category (currently you are using the index) -  may be worth setting key to review.review id and seeing if it fixes it?